### ISSUE TITLE
RemoveContentTypeFromList.cs does not remove from newly created list

### ISSUE THREAD

**Issue by brbarnett** [OfficeDev/PnP#625]
New-SPOList -Title "Events" -Template Events -QuickLaunchOptions off
Add-SPOContentTypeToList -List "Events" -ContentType "Custom Event Type" -DefaultContentType
Remove-SPOContentTypeFromList -List "Events" -ContentType "Event"

The New-* creates properly, Add-* adds the new custom type as default.  Remove-*, however, seems to do nothing.  The existing type is still in the list as Visible

**Comment by HappySolutions** [comment]
Hi. I just tested to run your your commands without any modifications after creating a content type manually called "Custom Event Type" that inherits from "Event". It seems to be working fine (see screenshot of list settings page after running the commands).

Have you tried running the commands on a clean site and list (to make sure there is nothing else causing the problem?).

![image](https://cloud.githubusercontent.com/assets/8503139/7018508/ace586f0-dd05-11e4-90a3-a1124d46fbdb.png)

**Comment by erwinvanhunen** [comment]
@brbarnett Any update on this issue? Where you able to reproduce it on a clean site?

**Comment by MikeBass13** [comment]
I am able to replicate this:
$web = Get-SPOWeb -Identity "Subweb"
New-SPOList -Title "Test" -Url "Test" -Template "Custom" -Web $web
Add-SPOContentTypeToList -List "Test" -ContentType "Issue" -DefaultContentType -Web $web
Remove-SPOContentTypeFromList -List "Test" -ContentType "Item" -Web $web

**Comment by patrick-rodgers** [comment]
Reopening based on @MikeBass13 's note on repro - we will take a look.

**Comment by erwinvanhunen** [comment]
I was able to reproduce the issue with Mike's repro description. It happened in situations where the CT was located in the rootweb, where it was added to a list in a subweb. When removing the content type internally a reference to the content type was made, e.g. it was looked up, but only in the current web. The cmdlet didn't find it there and the effect that no content type was removed.

I fixed it with PR https://github.com/OfficeDev/PnP-PowerShell/pull/222 in the PnP PowerShell repro.
